### PR TITLE
add metrics third-party custom info

### LINKED ISSUE #2798: [GetGVKsFromAddToSchem] Identify better ways to handling metrics filter for third-party API's and/or CRD which are cluster scoped.
## Feature Request

[user-guide: default-metrics-exported-with-3rd-party-resource](https://github.com/operator-framework/operator-sdk/blob/master/doc/user-guide.md#default-metrics-exported-with-3rd-party-resource)

Current documentation has gaps in understanding about the implementation to handle third party API schemas and their dependent schemas not registered with the manager. Also, I suspect there is an issue with registered schemes that also failed to be filtered for the OCP environment.

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
Suggested solution [#2794 
2794](https://github.com/operator-framework/operator-sdk/pull/2794)

Added variadic string param to GetGVKsFromAddToScheme for customkinds
it will allow to filter the custom third party or ocp schemas by passing
a list of kinds.

**Purpose: Identify a solution that can make it easier either by improving the documentation or  implementation.**

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description of the change:**
- Add info over how to customize the metrics when third-party APIS are added in the project
- Fix website doc which was outdated. 

Closes #2798
Related to :#2811

### CODE REVIEW

**Comment by camilamacedo86** [comment]
Agree @joelanford I will move it. Tks for the review.

**Comment by camilamacedo86** [comment]
Hi @joelanford, 

It was moved to the Monitoring/Metrics doc which shows more appropriated. Let me know if you are ok with now.

File: website/content/en/docs/golang/metrics/operator-sdk-monitoring.md [hunk 1]
```diff
@@ -87,6 +87,76 @@ By default operator will expose info metrics based on the number of the current
 
 The operator uses [Prometheus][prometheus] to expose a number of metrics by default. In order to expose custom metrics they have to be registered with the `Registry` object. An example can be found in the [kubebuilder book][kubebuilder].
 
+### With 3rd party resource
```
Comment (asmacdo) [inline website/content/en/docs/golang/metrics/operator-sdk-monitoring.md:90]: Is this related to Prometheus?

Comment (camilamacedo86) [reply website/content/en/docs/golang/metrics/operator-sdk-monitoring.md:90]: Are you asking if `3rd party resources` are related to Prometheus? 
No. 3rd party resources are any resource which is not from Kubernetes API.

Comment (camilamacedo86) [reply website/content/en/docs/golang/metrics/operator-sdk-monitoring.md]: Was added further information here. Please, let us know if it is clear for you now.

Comment (asmacdo) [reply website/content/en/docs/golang/metrics/operator-sdk-monitoring.md]: The reason I asked is that your other PR changes the title of this file. https://github.com/operator-framework/operator-sdk/pull/2821#pullrequestreview-391592131

If this isn't related to Prometheus, I dont think this belongs here.

Comment (camilamacedo86) [reply website/content/en/docs/golang/metrics/operator-sdk-monitoring.md]: The metrics are exported to Prometheus. If you check this doc you will see that it is speaking over this specific method as well above.

**Comment by camilamacedo86** [comment]
Closing this one since it will be affected by the integration with Kubebuilder. For further information see the comment : https://github.com/operator-framework/operator-sdk/issues/2798#issuecomment-613543951
